irc/server.go

```diff
@@ -497,6 +497,10 @@ func (server *Server) applyConfig(config *Config) (err error) {
 			return fmt.Errorf("Casemapping cannot be changed after launching the server, rehash aborted")
 		} else if oldConfig.Accounts.Multiclient.AlwaysOn != config.Accounts.Multiclient.AlwaysOn {
 			return fmt.Errorf("Default always-on setting cannot be changed after launching the server, rehash aborted")
+		} else if oldConfig.Server.Relaying.Enabled != config.Server.Relaying.Enabled {
```

Comment:
You could compare the entire config block by value (`oldConfig.Server.Relaying != config.Server.Relaying`).

Reply:
I think we can allow the `available-to-chanops` option to be changed on the fly without causing any issues, that's why it's checking those two options and not the whole block currently